flit-client/src/components/DialogLine/index.tsx

```diff
@@ -20,18 +23,26 @@ function makeTimestamp(stamp: Date):string {
 }
 
 export default function render(props: Props) {
+    let classNames:string = props.own ? "name ownName": "name"
+
+    console.log(props.user.avatar)
+    let avatar:string = props.user.avatar && props.user.avatar !== undefined && props.user.avatar !== "" ? `${avatarBase}/${props.user.avatar}` : `${avatarBase}/face.svg` 
```

Comment:
This is a bit long for a ternary one-liner.

I'd consider replacing it with:
```
const defaultAvatar = `${avatarBase}/face.svg`;
const getAvatar = (props: Props) => {
  if (props.user == null || props.user.avatar == null || props.user.avatar == "") {
    return defaultAvatar;
  }
  return props.user.avatar;
}
```

```
const avatar = getAvatar(props);
```

This is wordier, but it shuffles off most of the logic into a well-named function that you can ignore if you don't care about how avatar defaulting happens.